vaxrank/report.py

```diff
@@ -154,23 +163,14 @@ def compute_pdf_length_from_template_data(template_data):
     return length
 
 
-def make_html_report_from_ranked_vaccine_peptides(
-        ranked_variants_with_vaccine_peptides,
-        mhc_alleles,
-        variants,
-        bam_path,
-        html_report_path,
-        pdf_report_path=None):
-    template = JINJA_ENVIRONMENT.get_template('templates/template.html')
-    template_data = compute_template_data(
-        ranked_variants_with_vaccine_peptides,
-        mhc_alleles,
-        variants,
-        bam_path)
-    with open(html_report_path, "w") as f:
-        f.write(template.render(template_data))
+def make_pdf_report(
+        template_data,
+        pdf_report_path):
+    path = "%s.html" % uuid.uuid4()
```

Comment:
I tried to make it work with tempfile first, but pdfkit didn't seem to like reading from a temp HTML file. I'll play around with it some more, probably a weird permissions thing